third_party/amd/lib/TritonAMDGPUToLLVM/DecomposeUnsupportedConversions.cpp

```diff
@@ -58,74 +53,6 @@ static void promoteReduceOpResult(OpBuilder &builder, triton::ReduceOp op,
   }
 }
 
-static int getCvtOpLDSUsage(triton::gpu::ConvertLayoutOp &cvtOp) {
-  unsigned inVec = 0;
-  unsigned outVec = 0;
-  auto smemShape = triton::getScratchConfigForCvtLayout(cvtOp, inVec, outVec);
-  unsigned elems = getNumElements<unsigned>(smemShape);
-  auto srcType = cvtOp.getSrc().getType();
-  auto bytes =
-      isa<triton::PointerType>(srcType.getElementType())
-          ? elems * kPtrBitWidth / 8
```

Comment:
where kPtrBitWidth is defined ?

Reply:
It is defined here: https://github.com/triton-lang/triton/pull/3730/files#diff-69efd7149b566a254eabbb7b7808df841b5fb3e78f82d074bc26aa9369d4e4bfR19

I agree that it is not the cleanest solution, feel free to propose other place.